io.catenax.fleet.vehicles/2.0.0/Vehicles.ttl

```diff
@@ -0,0 +1,395 @@
+   samm:preferredName "Software category"@en ;
+   samm:description "Some OEMs bring in the software as complete package for all systems:\nTo identify this software: software category and software version is needed.\nSoftware category when this car was built."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "TZGH64738" .
+
+:softwareVersion a samm:Property ;
+   samm:preferredName "Software version"@en ;
+   samm:description "Some OEMs brings in the software as complete package for all systems:\nTo identify this software: software category and software version is needed.\nSoftware version when this car was built\n."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "3.4.9837.567" .
+
+:soldCountryCode a samm:Property ;
+   samm:preferredName "Sold country code"@en ;
+   samm:description "The country code in 3 letter format where this vehicle was sold."@en ;
+   samm:characteristic :CountryCodeTrait ;
+   samm:exampleValue "DEU" .
+
+:soldCountryGroup a samm:Property ;
+   samm:preferredName "Vehicle sold region"@en ;
+   samm:description "The region where this car was sold."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Europe" .
+
+:soldDate a samm:Property ;
+   samm:preferredName "Vehicle sold date:"@en ;
+   samm:description "Sold date of the vehicle = warranty start date for this vehicle"@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2018-02-03T00:00:00"^^xsd:dateTime .
+
+:steeringPos a samm:Property ;
+   samm:preferredName "Vehicle steering pos"@en ;
+   samm:description "Position of vehicle steering wheel, Left or right."@en ;
+   samm:characteristic :VehicleSteeringPos ;
+   samm:exampleValue "Left-Hand Drive" .
+
+:vehicleSeries a samm:Property ;
+   samm:preferredName "Vehicle series"@en ;
+   samm:description "vehicle series, normally one level above model. E.g. vehicle series =\"Golf\", vehicle model=\"Golf VIII\""@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Golf" .
+
+:wmiCode a samm:Property ;
+   samm:preferredName "WMI Code"@en ;
+   samm:description "The world manufacturer index are the first 3 letters of the VIN. IT describes a code for the vehicle manufacturer. Also see the description from NTHSA."@en ;
+   samm:see <https://vpic.nhtsa.dot.gov/> ;
+   samm:characteristic :OemShortNameTrait ;
+   samm:exampleValue "WVW" .
+
+:wmiNameNHTSA a samm:Property ;
+   samm:preferredName "WMI name according NHTSA"@en ;
+   samm:description "Name of the vehicle manufacturer according NHTSA: use https://vpic.nhtsa.dot.gov/api/vehicles/decodewmi/WVW?format=json - entry to be used is CommonName."@en ;
+   samm:see <https://vpic.nhtsa.dot.gov/> ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Volkswagen" .
+
+:engines a samm:Property ;
+   samm:preferredName "Engines"@en ;
+   samm:description "List of installed engines in the vehicle."@en ;
+   samm:characteristic :Engines .
+
+:equipments a samm:Property ;
```

Comment:
Property and the referenced Characteristic should not have the same name

Reply:
Changed to EquipmentsList